apache-rat-core/src/main/java/org/apache/rat/config/exclusion/fileProcessors/AbstractFileProcessorBuilder.java

```diff
@@ -151,7 +161,7 @@ protected MatcherSet process(final Consumer<MatcherSet> matcherSetConsumer, fina
         ExclusionUtils.asIterator(documentName.asFile(), commentFilter)
                 .map(entry -> modifyEntry(matcherSetConsumer, documentName, entry).orElse(null))
                 .filter(Objects::nonNull)
-                .map(entry -> ExclusionUtils.qualifyPattern(documentName, entry))
+                .map(entry -> ExclusionUtils.qualifyPattern(root, entry))
```

Comment:
this used to assume the pattern is relative to the location of the `.gitignore` file, which used to be true, but is not the case for the global gitignore.